Kickstarter-iOS/Views/Cells/BackingCell.swift

```diff
@@ -21,11 +37,32 @@ internal final class BackingCell: UITableViewCell, ValueCell {
 
     _ = self
       |> baseTableViewCellStyle()
-      |> BackingCell.lens.contentView.layoutMargins %~~ { layoutMargins, cell in
+      |> BackingCell.lens.contentView.layoutMargins %~~ { _, cell in
         cell.traitCollection.isRegularRegular
           ? .init(topBottom: Styles.grid(6), leftRight: Styles.grid(16))
-          : layoutMargins
+          : .init(topBottom: Styles.grid(3), leftRight: Styles.grid(2))
     }
+
+    _ = self.backingInfoButton
+      |> borderButtonStyle
+      |> UIButton.lens.title(forState: .normal) %~ { _ in Strings.backing_info_info_button() }
+      |> UIButton.lens.contentEdgeInsets .~ .init(topBottom: Styles.gridHalf(3),
+                                                  leftRight: Styles.gridHalf(5))
```

Comment:
any reason to adjust the insets instead of just using what the base button style gives us?

the base style also accounts for iPad by making the button a bit bigger.